test/testutil/storage/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 add_library(base_fs_test
     base_fs_test.hpp
     base_fs_test.cpp
-    )
+        ../../core/vm/actor/builtin/types/miner/monies_test_v2.cpp ../../core/vm/actor/builtin/types/miner/monies_test_v0.cpp ../../core/vm/actor/builtin/types/miner/monies_test_v3.cpp)
```

Comment:
remove

Reply:
If you have a problem with linking, you need to resolve it by adding necessary lib in `target_link_libraries`